distribution/archives/build.gradle

```diff
@@ -65,42 +70,42 @@ CopySpec archiveFiles(CopySpec modulesFiles, String distributionType, String pla
 distribution_archives {
   integTestZip {
     content {
-      archiveFiles(integTestModulesFiles, 'zip', null, 'x64', true)
```

Comment:
I'd personally prefer to keep it explicit here if it's a test distribution or not and keep that parameter here

Reply:
👍